### ISSUE TITLE
Use Grafana's Filesystem Usage metrics on Rancher's Disk Utilization dashboard for cluster metrics

### ISSUE THREAD

**Issue by dnoland1** [rancher/rancher#21254]
**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Steps to reproduce (least amount of steps as possible):**
Create a cluster and enable monitoring metrics. If your node has NFS shares mounted, the discrepancies will be more apparent.

View and compare the following:
1. Rancher Dashboard - Cluster Metrics - Disk Utilization
2. Grafana Dashboard - Filesystem Usage

**Result:**
For a given node, the disk utilization or filesystem usage % are different numbers. Appears that the Rancher Dashboard will use all mounts that are not rootfs devices, including network mounts. Grafana will only use devices that start with /dev/.

**Other details that may be helpful:**
According to metrics team, the following expressions are used.
Rancher:
```
(sum(node_filesystem_size_bytes{device!="rootfs"}) by (instance) - sum(node_filesystem_free_bytes{device!="rootfs"}) by (instance)) / sum(node_filesystem_size_bytes{device!="rootfs"})  by (instance)
```
Grafana:
```
sum (container_fs_usage_bytes{device=~"^/dev/.*$",id="/"}) by (node) / sum (container_fs_limit_bytes{device=~"^/dev/.*$",id="/"}) by (node) * 100
```
Enhancement request is to use the expression (or something similar) in Rancher that is used in Grafana. What Grafana displays is likely more in line with what an end user would expect.

Appears there's also a similar issue with memory metrics, see https://github.com/rancher/rancher/issues/18827

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
v2.2.4
- Installation option (single install/HA):
NA

**Comment by jiaqiluo** [comment]
The bug is reproduced in Rancher:v2.2.6

Steps:
- run Rancher:v2.2.6 container
- add an RKE single-node cluster
- enable the cluster monitoring
- mount an NFS shared disk to the node
```
> df -H
167.71.106.250:/nfs   84G  1.6G   82G   2% /nfs 
```
- view and compare the following metrics:
```
- Rancher Dashboard - Cluster Metrics - Disk Utilization
- Grafana Dashboard - Filesystem Usage
```

Result:
- in the Rancher dashboard one, the value drops from ~11% to ~5% after mounting the NFS shared disk
<img width="446" alt="226-1" src="https://user-images.githubusercontent.com/6218999/62331185-ff5a7a80-b46f-11e9-9485-01d5dee24b3d.png">
- in the Grafana Dashboard one, the value has no changes
<img width="630" alt="226-2" src="https://user-images.githubusercontent.com/6218999/62331205-100af080-b470-11e9-9e66-0fdddbf4f1bc.png">

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rnacher: master-head a8420b93

The same steps as the previous comment.

Results:
- the values in both metrics still are the same without changes 

<img width="454" alt="Screen Shot 2019-08-01 at 3 23 35 PM" src="https://user-images.githubusercontent.com/6218999/62331328-7b54c280-b470-11e9-865d-ccf4dea5fd74.png">

<img width="628" alt="Screen Shot 2019-08-01 at 3 23 52 PM" src="https://user-images.githubusercontent.com/6218999/62331334-7e4fb300-b470-11e9-9e30-36d0ccbd9fa7.png">
